lib/suma/tasks/bootstrap.rb

```diff
@@ -390,14 +390,14 @@ def setup_farmers_market(market_name:, market_address:, logo:, hero:, offering_p
     )
     create_product(
       vendor:,
-      name_en: "$2 in #{market_name} Vouchers",
-      name_es: "$2 en Cupones de #{market_name}",
+      name_en: "#{market_name} 1 to 1 Voucher Match",
+      name_es: "#{market_name} 1 a 1 de Cupones Igualados",
       description_en: "The suma voucher is a food special where suma works with you to buy down the price of vouchers for fresh and packaged food at #{market_name}. suma will match you 1:1 up to a $30 total (you load $15, suma matches with $15). You cannot use these vouchers for alcohol or hot prepared foods.",
       description_es: "El cupón de suma es un especial de alimentos en el que suma trabaja con usted para reducir el precio de los cupones para alimentos frescos y envasados en #{market_name}. suma te igualara 1:1 hasta un total de $30 (tu agregas $15, suma agrega $15 de créditos). No puede utilizar estos cupones para bebidas alcohólicas o comidas preparadas calientes.",
       our_cost: Money.new(200),
       vendor_service_categories: [fm2023_match_category],
       logo:,
-      max_quantity_per_order: 500,
+      max_quantity_per_order: 15,
```

Comment:
This needs to be a UI change, not a data change. This means we can't purchase more than 15 for an offering/user.